### PR TITLE
Add jitstress x jitstressregs

### PR DESCRIPTION
**Pull request by sejongoh**

### CODE REVIEW

**Comment by sejongoh** [comment]
@dotnet-bot test ci

**Comment by sejongoh** [comment]
@mmitche @RussKeldorph Could you please review the changes?

File: netci.groovy [hunk 1]
```diff
@@ -209,6 +216,13 @@ def static addTriggers(def job, def isPR, def architecture, def os, def configur
             case 'forcerelocs':
             case 'jitstress1':
             case 'jitstress2':   
+            case 'js2_jsr1':
```
Comment (RussKeldorph) [inline netci.groovy]: Is there a reason (like path length) that you abbreviated?  If not, I think it should be spelled out.

**Comment by RussKeldorph** [comment]
Need to squash commits

File: netci.groovy (hunk 1 above)
Comment (sejongoh) [inline netci.groovy]: I just wanted to make sure there is no path length problem and check functionality before using non-abbreviated names. If you think that using long ones in this checkin is better, I will update them.

Comment (sejongoh) [inline netci.groovy]: I think that I can check whether the long names are fine before checkin. I will update the names if there is no problem.

Comment (mmitche) [inline netci.groovy]: @sejongoh The shortener has been installed, so you can change the names if you want.

Comment (sejongoh) [inline netci.groovy]: Thanks! I will change the names.

**Comment by sejongoh** [comment]
@RussKeldorph The names are updated.

File: netci.groovy [hunk 2]
```diff
@@ -44,6 +44,13 @@ class Constants {
...
                'jitstressregs0x80' : ['COMPlus_JitStressRegs' : '0x80'],
+               'jitstress2_jitstressregs1'    : ['COMPlus_JitStress' : '2', 'COMPlus_JitStressRegs' : '1'],
+               'jitstress2_jitstressregs2'    : ['COMPlus_JitStress' : '2', 'COMPlus_JitStressRegs' : '2'],
+               'jitstress2_jitstressregs3'    : ['COMPlus_JitStress' : '2', 'COMPlus_JitStressRegs' : '3'],
+               'jitstress2_jitstressregs4'    : ['COMPlus_JitStress' : '2', 'COMPlus_JitStressRegs' : '4'],
+               'jitstress2_jitstressregs8'    : ['COMPlus_JitStress' : '2', 'COMPlus_JitStressRegs' : '8'],
+               'jitstress2_jitstressregs0x10' : ['COMPlus_JitStress' : '2', 'COMPlus_JitStressRegs' : '0x10'],
+               'jitstress2_jitstressregs0x80' : ['COMPlus_JitStress' : '2', 'COMPlus_JitStressRegs' : '0x80'],
                'fx' : ['' : ''], // corefx baseline
                'fxjs1' : ['COMPlus_JitStress' : '1'],
```
Comment (RussKeldorph) [inline netci.groovy]: I missed that these were abbreviated before.  We should fix these eventually as well, though I can't require it in this change.

**Comment by RussKeldorph** [comment]
LGTM

**Pull request merged**